src/components/screens/ScoreSetCreator.vue

```diff
@@ -326,6 +326,47 @@
                       </span>
                     </div>
                   </div>
+                  <div class="mavedb-wizard-row">
+                    <div class="mavedb-wizard-help">
+                      <label>
+                        An optional list of keywords that describe the experimental design of the score set. Score sets can be searched by keyword.
+                      </label>
+                      <div class="mavedb-help-small">
+                        Examples: SGE, barcode sequencing, MPRA, complementation
+                      </div>
+                    </div>
+                    <div class="mavedb-wizard-content field">
+                      <span class="p-float-label">
+                        <AutoComplete
+                            ref="contributorsInput"
+                            v-model="contributors"
```

Comment:
We should consider adding the contributors to the list of things auto-populated by experiment. Alan noted that most of the time are the same between the two items so it'd be nice for that to be the default. 

We'd just need to add something like
`this.contributors = event.value.contributors` in `populateExperimentMetadata`.

Reply:
It's populated now from either the experiment or the superseded score set.